### ISSUE TITLE
Fix inspector warnings

### ISSUE THREAD

**Issue by danielrbradley** [pulumi/pulumi-awsx#848]
The following warning text is emmitted on provider instantiation, which doesn't actually cause a failure but is not helpful output.

```
    error: [runtime] Error [ERR_INSPECTOR_NOT_AVAILABLE]: Inspector is not available
        at new NodeError (node:internal/errors:372:5)
        at node:inspector:23:9
        at NativeModule.compileForInternalLoader (node:internal/bootstrap/loaders:312:7)
        at NativeModule.compileForPublicLoader (node:internal/bootstrap/loaders:252:10)
        at loadNativeModule (node:internal/modules/cjs/helpers:49:9)
        at Function.Module._load (node:internal/modules/cjs/loader:804:15)
        at Module.require (node:internal/modules/cjs/loader:999:19)
        at Module.require (pkg/prelude/bootstrap.js:1812:31)
        at require (node:internal/modules/cjs/helpers:102:18)
        at /snapshot/bin/node_modules/@pulumi/pulumi/runtime/closure/v8Hooks.js:44:62
```

**Comment by danielrbradley** [comment]
One option is to build the nodejs binary ourselves with the --inspect-brk flag set: https://github.com/vercel/pkg/issues/93#issuecomment-301210543

**Comment by danielrbradley** [comment]
A better option would be to fix the side-effect of importing `@pulumi/pulumi` which causes the inspector to be pre-loaded on import, even if unused.

**Comment by danielrbradley** [comment]
This is now resolved in `1.0.0-beta.7` by using the latest unreleased version of `@pulumi/pulumi`. This should be change to version 3.34 once it's released.
